ppr-ui/src/views/dashboard.vue

```diff
@@ -1,42 +1,82 @@
 <template>
   <v-container fluid class="pa-10">
-    <v-row no-gutters>
-      <v-col cols="12" class="search-title">
-        <b>Search</b>
-      </v-col>
-    </v-row>
-    <v-row no-gutters>
-      <v-col cols="12" class="search-info pt-2">
-        <span>
-          Select a search category and then enter a value to search.
-          <b>Note:</b>
-          Each search incurs a fee (including searches that return no results).
-        </span>
+    <v-row>
+      <v-col>
+        <v-row no-gutters>
+          <v-col cols="12" class="search-title">
+            <b>Search</b>
+          </v-col>
+        </v-row>
+        <v-row no-gutters>
+          <v-col cols="12" class="search-info pt-2">
+            <span>
+              Select a search category and then enter a value to search.
+              <b>Note:</b>
+              Each search incurs a fee (including searches that return no results).
+            </span>
+          </v-col>
+        </v-row>
+        <v-row>
+          <search @search-error="emitError"
+                  @search-data="setSearchResults">
+          </search>
+        </v-row>
       </v-col>
     </v-row>
     <v-row>
-      <search @search-error="emitError" @search-data="setResults">
-      </search>
+      <v-col>
+        <v-row no-gutters>
+          <v-col cols="12" class="search-title">
+            <b>Search Results</b> <span id="search-time"> {{ searchTime }}</span>
+          </v-col>
+        </v-row>
+        <v-row no-gutters>
+          <v-col cols="12" class="search-info pt-4">
+            <span v-if="!getSearchResults">
+              Your search results will display below.
+            </span>
+            <span v-else-if="totalResultsLength !== 0">
+              Select the registrations you want to download in a printable PDF.
+              Your search results, selected registrations, and PDF are automatically saved to My Searches.
+            </span>
+            <span v-else>
+              Your search results and PDF are automatically saved to My Searches.
+            </span>
+          </v-col>
+        </v-row>
+        <v-row v-if="getSearchResults">
+          <result :data="getSearchResults">
```

Comment:
You can self close components if you prefer. ie `<result :data="getSearchResults" />

Reply:
ah okay sounds good will update as it looks cleaner that way